### PR TITLE
Added settings page

### PR DESCRIPTION
**Pull request by vladdezhniuk**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/Currency/list.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+              </CurrencySelectButton>
+            </li>
+          ))}
+      </CurrenciesList>
+    </>
+  )
+}
+
+const mapStateToProps = (state) => ({
+  currencies: [
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/Currency/list.tsx]: why this is a part of `mapStateToProps `? It's just a constant which is not connected to redux right?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/Language/list.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import { selectors } from 'data'
+import { languagesSortedByName } from 'services/locales'
+
+import { CurrecyItemWrapper, CurrenciesList, CurrencySelectButton } from '../Currency/list'
+
+const List = (props) => {
+  const selectCurrency = (value) => {
+    if (value === props.language) return
+    props.input.onChange(value)
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/Language/list.tsx:16]: I don't see `input` in props list. How does this work?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/index.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import Currency from './Currency'
+import Language from './Language'
+import { Overal } from './Overal'
+
+export const General = (props) => {
+  const { path } = props.match
+
+  return (
+    <>
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/index.tsx]: do we need empty tag here?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/Info/index.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  public label: string
+
+  public id: string
+
+  constructor(path, label, id) {
+    this.path = path
+    this.label = label
+    this.id = id
+  }
+}
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/Info/index.tsx:35]: what about rewriting this class like this

```
class AboutLink {
  constructor(
    public path: string, 
    public label: string, 
    public id: string
  ) {}
}
```

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/Networks/index.tsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+  public isConnected: boolean
+
+  constructor(messageId: string, icon: any, label: string, isConnected: boolean) {
+    this.messageId = messageId
+    this.icon = icon
+    this.label = label
+    this.isConnected = isConnected
+  }
+}
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/Networks/index.tsx:59]: same about class

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/Overal/index.tsx [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+      </SettingsList>
+      <LogoutButton onClick={logout}>
+        <FormattedMessage id='layouts.wallet.header.Sign Out' defaultMessage='Sign Out' />
+      </LogoutButton>
+    </>
+  )
+}
+
+const mapDispatchToProps = (dispatch) => ({
+  routerActions: bindActionCreators(actions.router, dispatch),
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/Overal/index.tsx]: is this ever used?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/index.tsx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+  public path: string
+
+  constructor(id: string, label: string, path: string, icon?: any) {
+    this.id = id
+    this.label = label
+    this.path = path
+    this.icon = icon
+  }
+}
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/index.tsx:43]: same about class

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/Language/list.tsx (hunk 2 above)
Comment (vladdezhniuk) [reply packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/Language/list.tsx:16]: It appears after connection to redux form by unique key

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/Info/index.tsx (hunk 4 above)
Comment (vladdezhniuk) [reply packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/Info/index.tsx:35]: No, I know this syntax, but their lint will not let me use it

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/Networks/index.tsx (hunk 5 above)
Comment (vladdezhniuk) [reply packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/Networks/index.tsx:59]: Same

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/index.tsx (hunk 7 above)
Comment (vladdezhniuk) [reply packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/index.tsx:43]: Same answer

**Review by VitaliiShpital — APPROVED** [review]

**Pull request merged**